src/components/details.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+
+const Details = () => {
+  const { id } = useParams();
+  const push = useNavigate();
+  const animals = useSelector((state) => state.animals);
+  const [animal, setAnimal] = useState({});
+  useEffect(() => {
+    animals.forEach((an) => {
+      if (parseInt(an.id, 10) === parseInt(id, 10)) {
+        setAnimal(an);
+      }
+    });
+  }, []);
+  return (
+    <div className="Details">
+      <button type="button" onClick={() => push('/')}>
+        <i className="fa fa-arrow-left" aria-hidden="true" />
+        Home
+      </button>
```

Comment:
- [ ] As indicated in the design the **`Back`** button (<) should be placed in the design, the back arrow for the **Details** page should be on the header. Kindly fix that.

  ![nemwel-back-edit](https://user-images.githubusercontent.com/36542986/164275400-6e14de8d-f9f7-4204-8eeb-8a059e0e7eb7.jpg)

Reply:
Hello @TedLivist thank you for this review.
I have effected this change